src/services/meteorService.js

```diff
@@ -0,0 +1,58 @@
+const nasaRepository = require("../repositories/nasaRepository");
+const Exception = require("../utils/exception");
+const dateUtils = require("../utils/dateUtils");
+
+const getMeteorsData = async ({ date, count, wereDangerousMeteors }) => {
+    const startDate = dateUtils.getStartDateOrDefault(date);
+    const endDate = dateUtils.getEndDateOrDefault(date, startDate);
```

Comment:
date can be range of dates
I updated task description

Reply:
ok